crates/stackable-operator/src/commons/listener.rs

```diff
@@ -100,14 +123,28 @@ pub struct ListenerSpec {
     pub ports: Option<Vec<ListenerPort>>,
 
     /// Whether incoming traffic should also be directed to Pods that are not `Ready`.
-    #[schemars(default = "Self::default_publish_not_ready_addresses")]
+    #[serde(default = "ListenerSpec::default_publish_not_ready_addresses")]
     pub publish_not_ready_addresses: Option<bool>,
+
+    /// `externalTrafficPolicy` that should be set on the [`Service`] object.
+    ///
+    /// The default is `Local` (in contrast to `Cluster`), as we try pretty hard to shove traffic onto the right node,
+    /// and we should keep testing that as the primary configuration. Cluster is a fallback option for providers that
+    /// break Local mode (IONOS so far).
```

Comment:
Probably want to reword this if we're putting it in user facing docs.

Reply:
Perhaps:

```
as we aim to direct traffic to the nearest node
````

_Other edits needed, but I didn't get to them_

Reply:
"to a node running the workload" perhaps.

Reply:
Please have a look at f59ff2dfe796d91e5dd18fe747faf31266d2930b